app/controllers/podcasts_controller.rb

```diff
@@ -37,10 +37,16 @@ def show
   # GET /podcasts/new
   def new
     @podcast = Podcast.new(podcast_params)
+    @podcast.set_default_feed
+
+    # TODO: get the default account from ID somehow
+    @podcast.prx_account_uri = helpers.podcast_account_name_options(@podcast).first.last
+    @podcast.clear_attribute_changes(%i[prx_account_uri])
```

Comment:
I needed to set the initial selected account_uri on `#new` - to some account that this user has edit access to.  And then mark it as unchanged, so you don't get the orange outline right away.

But would be much better to somehow get their default account from ID.  Someday...